config/vufind/config.ini

```diff
@@ -2110,6 +2110,10 @@ treeSearchLimit = 100
 ;EIT = access.EITModule
 ;Primo = access.PrimoModule
 
+
+[SearchTabsDetails]
```

Comment:
Would SearchTabsSettings be more open-ended if we want to add more in the future? Interested in whether @EreMaijala has any thoughts/preferences. Also, before we merge this, we should be sure to add some comments describing the purpose of the new section and setting.

Reply:
I agree that SearchTabsSettings would make sense. I'd also make the bool setting `... = true` instead of `... = 1`.

I think we're slowly approaching the point where moving search tab configuration to a separate config file would make sense, but this change won't mandate a switch.

Reply:
Ok; I changed it to SearchTabsSettings.